packages/test/test-pairwise-generator/src/index.ts

```diff
@@ -5,15 +5,24 @@
 
 import random from "random-js";
 
-// converts all properties of an object to arrays of the
-// properties potential values. This will be used by generatePairwiseOptions
-// to compute original objects that contain pairwise combinations
-// of all property values
+/**
+ * Converts all properties of an object to arrays of the
+ *	properties potential values. This will be used by generatePairwiseOptions
+ *	to compute original objects that contain pairwise combinations of all property values
+ *	@internal
```

Comment:
Alignment nit; tabs vs spaces
```suggestion
 * Converts all properties of an object to arrays of the
 * properties potential values. This will be used by generatePairwiseOptions
 * to compute original objects that contain pairwise combinations of all property values
 * @internal
```